src/modules/auth/screens/InvitationScreen.js

```diff
@@ -169,6 +170,12 @@ export default function InvitationScreen(props) {
 									</div>
 								</CardHeader>
 								<CardBody>
+									<FlowbiteIllustration
```

Comment:
You should wrap the Flowbite component to some div, where you specify the width/height

Reply:
@JanMajer86 understand, but I dont think it is desired